### DISCUSSION TITLE
ERR_SSL_PROTOCOL_ERROR

### DISCUSSION THREAD

**Discussion by Alan7921** [PaddlePaddle/PaddleOCR#13913] [Q&A]
在下载模型 https://paddleocr.bj.bcebos.com/PP-OCRv4/chinese/ch_PP-OCRv4_rec_infer.tar 时遇到了SSL error，除了该模型几乎所有模型的url都遇到了SSL error

**Comment by GreatV** [comment]
没有问题呀，检查一下你自己本地的网络环境吧
